Source/DafnyCore/DafnyOptions.cs

```diff
@@ -106,7 +106,7 @@ public class DafnyOptions : Bpl.CommandLineOptions {
         argumentName: "Everything|DllEmbed|NoIncludes|NoGhost",
         defaultValue: PrintModes.Everything);
 
-      RegisterLegacyUi(CommonOptionBag.DefaultFunctionOpacity, ParseDefaultFunctionOpacity, "Language feature selection", "defaultFunctionOpacity", null, CommonOptionBag.DefaultFunctionOpacityOptions.Transparent);
+      RegisterLegacyUi(CommonOptionBag.DefaultFunctionOpacity, ParseDefaultFunctionOpacity, "Language feature selection", "defaultFunctionOpacity", null, CommonOptionBag.DefaultFunctionOpacityOptions.AutoRevealDependencies);
```

Comment:
I think it should still say Transparent here, right?

Reply:
You're right, I changed it to test something, then forgot to change back before merging!